Traceback (most recent call last):
  File "/home/sadbhawna/Desktop/cs6910_assignment1/train.py", line 146, in <module>
    train_arguments(args) # python train.py --dataset mnist --epochs 100 -nhl 3 -sz 64
    ^^^^^^^^^^^^^^^^^^^^^
  File "/home/sadbhawna/Desktop/cs6910_assignment1/train.py", line 122, in train_arguments
    fit(layer_architecture, X_train, Y_train, X_val, Y_val, X_test, Y_test, epochs=args.epochs, activation=args.activation, loss = args.loss, optimizer=args.optimizer, weight_ini = args.weight_init,
  File "/home/sadbhawna/Desktop/cs6910_assignment1/train.py", line 41, in fit
    Nadam(layer_architecture, X_train, Y_train, X_val, Y_val, X_test, Y_test, epochs=epochs, activation=activation, loss=loss, weight_ini = weight_ini, learning_rate=learning_rate, beta1=beta1, beta2=beta2, batch_size=batch_size,
  File "/home/sadbhawna/Desktop/cs6910_assignment1/optimizers.py", line 345, in Nadam
    nn = Feedforward_NeuralNetwork(layer_architecture, activation, weight_ini, loss)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sadbhawna/Desktop/cs6910_assignment1/Feedforward_Neural_Network.py", line 8, in __init__
    self.activation = self.find_activation_functions(activation)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sadbhawna/Desktop/cs6910_assignment1/Feedforward_Neural_Network.py", line 25, in find_activation_functions
    raise ValueError("Unsupported activation function")
ValueError: Unsupported activation function